its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/Tests.java

```diff
@@ -72,7 +72,7 @@ private static Orchestrator createOrchestrator() {
     }
     // The number of results depends on the XML plugin version. Since not all plugin versions are compatible with
     // all SonarQube versions, we will run tests with XML plugin on the latest SQ version.
-    if (version.equals("LATEST_RELEASE")) {
+    if (version.equals("LATEST_RELEASE") || version.equals("DEV")) {
```

Comment:
Why this was needed?
Before updating the ORCHESTRATOR, the XML plugin was not added when running against DEV instances.

Reply:
In discussion with Costin, we believe that the latest release of the xml plugin is now picking up app.config when it was not previously.

The tests themselves [run only on 10.8](https://github.com/SonarSource/sonar-scanner-msbuild/blob/2cb7d7a10931b4e80dd79a4f902d55635dbceb36/its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/ScannerMSBuildTest.java#L348) or higher.

Installing the XML plugin on LATEST but not DEV results in different base scenarios, with different results.